robonomics/frame/datalog/Cargo.toml

```diff
@@ -21,6 +22,7 @@ pallet-timestamp = { git = "https://github.com/paritytech/substrate", branch = "
 
 [features]
 default = ["std"]
+use_twox = []
```

Comment:
Is it used for enabling IPCI backward compatibility? Let's name it `storage-v0` and disable it by default. On the IPCI chain we could turn it on for backward compatibility. But fresh chain will be launched without legacy structures in the state.

Reply:
Yes, this feature requires for IPCI compatibility.  IPCI uses black2 hasher while  robomonics uses xxhash.